settings

```diff
@@ -23,3 +23,5 @@ export CLIENT_VERSION="0.8-SNAPSHOT"
 
 # Name of the release package
 export PACKAGE_NAME="@kubernetes/node-client"
+
+export OPENAPI_GENERATOR_COMMIT=v4.1.0
```

Comment:
Should we be using `export KUBERNETES_BRANCH="release-1.13"` now that 1.15 is out

Reply:
Ah, yeah, I think so.  I'll update and regenerate.